react/ListGroup.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const ListGroup: React.FC = ({ children }) => {
+  return (
+    <div className="nr5 nr0-ns bt bb b--muted-4">
+      <div className="nl5 nl0-ns pa0" role="group">
```

Comment:
A `div` has no default padding so this class `pa0` might not be useful.

Also can't this inner div be merged with its parent?

Reply:
And why the negative margin?

Reply:
nice catch! I used to have a `li` element here, so that's why I used the `pa0` class

Reply:
the negative margin is tricky, but basically its so I can have the width of the group to be 100% on mobile, but to be limited to the parent when not

Reply:
I'll see about merging these two divs

Reply:
the difference of the two divs being separated is subtle, but isn't what I want :-/

separated:

![image](https://user-images.githubusercontent.com/10223856/79999001-c5b02080-8491-11ea-94e2-db1103ba1855.png)

merged:

![image](https://user-images.githubusercontent.com/10223856/79999122-ebd5c080-8491-11ea-9497-f9b25908d450.png)

the only difference is that the outer border must go all the way to the right, but need the margin on the left